File: cNew/sumeven.c
//sum of all even no 
#include<stdio.h>
void main()
{
    int i,n,sum=0;
    printf("enter n\n");
    scanf("%d",&n);
    // for(i=1;i<=n;i++)
    // {  
    //  if(n%2==0)
    //     {
    //         printf("even=%d\n",i);
    //         sum=sum+i;
    //     }
    // }
    // printf("sum of even no = \n",sum);
        
        for(i=2;i<=n;i+=2)
        {
          printf("even no = %d\n",i);
          sum=sum+i;
        }
        printf("sum od even no = %d\n",sum);
}

File: java/Java programs/sumeven.java

import java.util.Scanner;

public class sumeven 
{ public static void main(String[] args) 
    {   Scanner sc=new Scanner(System.in);
        int n,i;
        System.out.println("enter the number of array element");

         n= sc.nextInt();
        int arr[]=  new int[n];
        System.out.println("enter the element of array");

       for(i=0;i<n;i++)
       {
        arr[i]=sc.nextInt();
       }
       int sum=0;

       for ( i = 0; i < n; i++) {
        if (arr[i] % 2 == 0) {
            sum += arr[i];
        }
    }
    System.out.println("sum of even numbers="+sum);


    
}
    
}
